### PR TITLE
Not all constants in `fcntl` are available on macos

### PR DESCRIPTION
**Pull request by sobolevn**
Let's try this combination

### CODE REVIEW

**Comment by AlexWaygood** [comment]
IIRC (please check), `if sys.platform == linux` means "only present on Ubuntu". I think if you want to say "only present on [grab-bag of miscellaneous linux systems]", you need to say `if sys.platform != "win32" and sys.platform != "darwin"`

File: stdlib/fcntl.pyi [hunk 1]
```diff
@@ -47,6 +47,9 @@ if sys.platform != "win32":
             F_OFD_GETLK: int
             F_OFD_SETLK: int
             F_OFD_SETLKW: int
+        if sys.version_inf >= (3, 10):
```
Comment (AlexWaygood) [inline stdlib/fcntl.pyi:50]:
```suggestion
        if sys.version_info >= (3, 10):
```

**Comment by sobolevn** [comment]
Maybe we should add some extra OS to the test? We can't do that natively, but I can add some docker-container tests 🤔 

I will add an issue for that.

**Comment by Akuli** [comment]
>IIRC (please check), if sys.platform == "linux" means (bizarrely) "only present on Ubuntu". 

This isn't correct. `if sys.platform == "linux"` is the correct check for "only present on Ubuntu", and also "present on all linux distros". A check like `if sys.platform != "win32" and sys.platform != "darwin"` will also be true on other posix systems than linux and macos, such as BSD and Solaris, so it isn't the right choice for linux-specific things.

**Comment by AlexWaygood** [comment]
> > IIRC (please check), if sys.platform == "linux" means (bizarrely) "only present on Ubuntu".
> 
> This isn't correct. `if sys.platform == "linux"` is the correct check for "only present on Ubuntu", and also "present on all linux distros". A check like `if sys.platform != "win32" and sys.platform != "darwin"` will also be true on other posix systems than linux and macos, such as BSD and Solaris, so it isn't the right choice for linux-specific things.

Thanks for the correction!

**Review by srittau — APPROVED** [review]
I didn't double check in depth, but LGTM.

**Pull request merged**